doc/src/quickstart/index.rst

```diff
@@ -485,6 +391,135 @@ which we can visualize with:
         --route_chan_width 100 \
         --analysis --disp on
 
+Manually Running VTR with ODIN II
+----------------------------------
+VTR includes a second synthesis tool, ODIN II. Below we explain how to run this alternative synthesis flow.
+
+Let's start by making a new directory for us to work in:
+
+.. code-block:: bash
+
+    > mkdir -p ~/vtr_work/quickstart/blink_manual
+    > cd ~/vtr_work/quickstart/blink_manual
+
+To synthesize your Verilog design with ODIN II, you need to build VTR with the following command:
+
+.. code-block:: bash
+
+    > make CMAKE_PARAMS="-DWITH_ODIN=on"
```

Comment:
This is confusing since the line above tells the user to change directory to `~/vtr_work/quickstart/blink_manual` and now you are saying to use the make command. This should be in the VTR directory. I would move the build instructions above the prior code block and provide the `cd` command to go to the correct directory.